bedevere/stage.py

```diff
@@ -222,6 +222,9 @@ class Blocker(enum.Enum):
             thanks = BORING_THANKS
         comment = ACK.format(greeting=thanks, core_devs=core_devs)
         await gh.post(issue["comments_url"], data={"body": comment})
+        reviewers_url = pr_url + '/requested_reviewers'
+        reviewers = [core_dev async for core_dev in core_dev_reviewers(gh, pr_url)]
+        await gh.post(reviewers_url, data={"reviewers": reviewers})
```

Comment:
Does `"team_reviewers": []` need to be provided in the `data`?

Reply:
I have not live-tested this, but my experience with the Github API Docs is that it is explicitly stated, when a JSON field is required, which is not the case here. [An example](https://developer.github.com/v3/repos/#input) is the `name` field of the "create repository" API.

Reply:
Would you mind live-testing this in a test repo to make sure? I'm basically trying to avoid Bedevere falling over and having to do an emergency PR. 😄 I'm happy to add a needless review somewhere to help give you data to play with.

Reply:
I confirmed that `"team_reviewers"` can be omitted.

I tested in real-life with the following snippet:
```
import asyncio
import aiohttp
from gidgethub.aiohttp import GitHubAPI
import gidgethub
import os
async def main():
    async with aiohttp.ClientSession() as session:
        gh = GitHubAPI(session, "miss-islington", oauth_token=os.getenv("GH_AUTH"))
        try:
            result = await gh.post('/repos/python/miss-islington/pulls/253/requested_reviewers',
                                   data={"reviewers": ["miss-islington"]})
        except gidgethub.BadRequest as exc:
            print(exc)
asyncio.run(main())
```

Result: https://github.com/python/miss-islington/pull/253#event-2477015012

Reply:
@brettcannon Is the live-testing by Marietta enough, or should I deploy my fork and test it on a test repository?

Reply:
@lysnikolaou I trust @Mariatta so it's enough 😄